.github/ISSUE_TEMPLATE/bug-report.md

```diff
@@ -0,0 +1,17 @@
+---
+name: Bug Report
+about: Create a report to help us improve
+title: "[Issue]"
+labels: 'T: defect'
+assignees: ''
+
+---
+
+**Describe the bug**
```

Comment:
How about "Bug Description" and "Steps to Reproduce" for grammatical consistency between headings?

Reply:
Seems reasonable